README.md

```diff
@@ -1,5 +1,7 @@
 # Gatling DemoStore Setup
 
+[![Workflow Status (main)](https://img.shields.io/github/actions/workflow/status/gatling/gatling-demostore/build.yml?branch=main&logo=github&style=for-the-badge)](https://github.com/gatling/gatling-demostore/actions?query=branch%3Amain)
```

Comment:
Didn't check that, but why not those generated by github?
https://github.com/gatling/gatling-demostore/actions/workflows/build.yml/badge.svg

[![Build](https://github.com/gatling/gatling-demostore/actions/workflows/build.yml/badge.svg)](https://github.com/gatling/gatling-demostore/actions/workflows/build.yml)

Reply:
We have some repo that uses shields, which does not support private repos. Find it prettier but I guess it would make more sense to use the same badge style everywhere.